src/yunohost/tools.py

```diff
@@ -474,23 +475,39 @@ def tools_update(ignore_apps=False, ignore_packages=False):
     # "packages" will list upgradable packages
     packages = []
     if not ignore_packages:
-        cache = apt.Cache()
 
         # Update APT cache
+        # LC_ALL=C is here to make sure the results are in english
+        command = "LC_ALL=C apt update"
+        # TODO : add @is_unit_operation to tools_update so that the
+        # debug output can be fetched when there's an issue...
+
+        # Filter boring message about "apt not having a stable CLI interface"
+        # Also keep track of wether or not we encountered a warning...
+        warnings = []
+        def is_legit_warning(m):
+            legit_warning = m.rstrip() and "apt does not have a stable CLI interface" not in m.rstrip()
```

Comment:
Shouldn't we use "apt-get" instead of "apt"? iirc apt is designed to be runed interactively not for scripting.